h2/src/main/org/h2/mvstore/MVMap.java

```diff
@@ -164,16 +164,6 @@ public final K firstKey() {
         return getFirstLast(true);
     }
 
-    /**
-     * Get the first key of this page.
-     *
-     * @param p the page
-     * @return the key, or null
-     */
-    public final K firstKey(Page<K,V> p) {
```

Comment:
Nice work!

Looks good, but why you removed this and the following methods? They are for the incomplete implementation of the sorted map.

Reply:
Never mind, the standard methods are preserved.